### PR TITLE
Feature/us1460 share js components

### PR DESCRIPTION
**Pull request by charlie-retzler**
@silbermm modularized the javascript libraries

### CODE REVIEW

File: crossroads.net/app/mp_tools/mpTools.run.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"use strict";
+(function() {
+    angular.module("crossroads.mptools").run( AppRun );
+
+    AppRun.$inject = ["$location", 'MPTools'];
+    //AppRun.$inject = ["Session", "$rootScope", "MESSAGES", "$http", "$log", "$state", "$timeout", "$location"];
```
Comment (silbermm) [inline crossroads.net/app/mp_tools/mpTools.run.js]: remove commented out code in this file

File: crossroads.net/core/app.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict()';
```
Comment (silbermm) [inline crossroads.net/core/app.config.js]: better to name this file the same as the module... i.e. `core.config.js` or `crossroads.core.config.js`

File: crossroads.net/core/app.controller.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 'use strict()';
 (function(){
 
-  angular.module("crossroads").controller('appController', AppController);
+  angular.module("crossroads.core").controller('appController', AppController);
```
Comment (silbermm) [inline crossroads.net/core/app.controller.js]: I think this controller should be converted to being called `CoreController` and the file renamed to `core.controller.js`. Of course if this happens, then the reference to  `appController` in the `index.html` file will need to be update as well.

File: crossroads.net/core/app.core.module.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+'use strict()';
```
Comment (silbermm) [inline crossroads.net/core/app.core.module.js]: whatever you decide to call the `core.config.js` file, this should follow the naming convention, i.e. `core.module.js` or `crossroads.core.module.js`

File: crossroads.net/core/app.run.js [hunk 5]
```diff
@@ -1,20 +1,17 @@
 ﻿"use strict";
```
Comment (silbermm) [inline crossroads.net/core/app.run.js]: `core.run.js` or `crossroads.core.run.js`

**Comment by charlie-retzler** [comment]
Updated code per @silbermm codereview

**Pull request merged**